lisa/features/gpu.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from enum import Enum
+from typing import Any
+
+from lisa.feature import Feature
+from lisa.operating_system import Linux, Redhat, Ubuntu
+from lisa.tools import Uname, Wget
+from lisa.util import LisaException
+
+FEATURE_NAME_GPU = "Gpu"
+
+
+class compute_sdk(Enum):
+    GRID = 0
+    CUDA = 1
+
+
+class Gpu(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
+        super().__init__(node, platform)
+        self._log = self._node.log
+
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_GPU
+
+    def _install_grid_driver(self, version: str) -> None:
+        self._log.info("Starting GRID driver installation")
+
+    def _install_cuda_driver(self, version: str = "CUDA_DRIVER") -> None:
+        self._log.info("Starting CUDA driver installation")
+        cuda_repo = ""
+        distro = Linux(self._node)
```

Comment:
why create an instance here? It should be like `os = self._node.os`, if a shortcut is needed. If it need the type, try `os = cast(Linux, self._node.os)`

Reply:
> why create an instance here? It should be like `os = self._node.os`, if a shortcut is needed. If it need the type, try `os = cast(Linux, self._node.os)`

Done